common.gradle

```diff
@@ -0,0 +1,56 @@
+//
```

Comment:
Is this build script used anywhere? I don't have actually used a common.gradle file before, reading about it, it seems that you need to put `apply from: '../common.gradle' ` to use it somewhere else?